.github/workflows/demo-server-down.yml

```diff
@@ -6,33 +6,46 @@ on:
     paths-ignore:
       - '**.md'
       - 'docs/**'
+  workflow_dispatch:
 
 env:
   B2C_APP_SUFFIX: universal-b2c
   B2B_APP_SUFFIX: universal-b2b
 
 jobs:
   RemoveDemoServer:
-    if: github.event.pull_request.head.repo.owner.login == 'intershop'
+    if: github.event.repository.full_name == 'intershop/intershop-pwa'
     runs-on: ubuntu-latest
     steps:
+      - name: Determine App Names
+        run: |
+          if [[ -n ${{ github.event.pull_request.head.ref }} ]]
```

Comment:
I guess you have to use double `[` in github workflows? Asking for a friend

Reply:
No, that's just the `[[` compound form in bash. I was used only to `test` and `[`.